### DISCUSSION TITLE
i have question

### DISCUSSION THREAD

**Discussion by sureface** [community/community#19348] [Other Feature Feedback, Questions, & Ideas]
why need a different achirments like yayo so on...

**Comment by melvincwng** [comment]
Hi @sureface, achievements are just for cosmetic purposes. They don't serve much purpose, other than acting as a tiny badge of honor, to be honest.

For a full list of all achievements, please refer to this link: https://github.com/Schweinepriester/github-profile-achievements

Hope this clarifies.

If this has helped you, kindly 'Mark as answer' so the rest know this issue has been addressed.

**Comment by cupid20103** [comment]
Profiles on GitHub give developers the opportunity to show off their favorite projects, contributions, and all the things that they choose to highlight in their profile README.
So if you had many achievements, you will look like a professional developer.

**Comment by radar-blip** [comment]
You dont need them. They are awarded to you after completing certain achievements. There is an option in settings to hide them if you dont want them to show.

**Comment by jfullstackdev** [comment]
latest update, you can now pick what you want to hide, unlike in the past, it was all or nothing

go to Achievements Tab, click the `Achievements` word, it is a link

![image](https://user-images.githubusercontent.com/47092464/204711684-e7c5ebb6-57f9-4fbf-b8fe-96c2182f41bb.png)

or this URL, it's same thing 

https://github.com/[username]?tab=achievements

you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711801-dde96c4d-5133-4201-ad8a-9b23b55ed5d2.png)

then, click any badge, hover left then you should be seeing like this 

![image](https://user-images.githubusercontent.com/47092464/204711940-2c5e8b4d-7580-4145-afc0-268f1739d8de.png)

then boom ... you now have the option to hide those badges that you want to hide
